recipes/llm-voice-assistant/dotnet/LLMVoiceAssistant/Program.cs

```diff
@@ -0,0 +1,751 @@
+                    case "--picollm_frequency_penalty":
+                        if (++argIndex < args.Length)
+                        {
+                            config.PicollmFrequencyPenalty = float.Parse(args[argIndex++]);
+                        }
+                        break;
+                    case "--picollm_temperature":
+                        if (++argIndex < args.Length)
+                        {
+                            config.PicollmTemperature = float.Parse(args[argIndex++]);
+                        }
+                        break;
+                    case "--picollm_top_p":
+                        if (++argIndex < args.Length)
+                        {
+                            config.PicollmTopP = float.Parse(args[argIndex++]);
+                        }
+                        break;
+                    case "--picollm_system_prompt":
+                        if (++argIndex < args.Length)
+                        {
+                            config.PicollmSystemPrompt = args[argIndex++];
+                        }
+                        break;
+                    case "--orca_warmup_sec":
+                        if (++argIndex < args.Length)
+                        {
+                            config.OrcaWarmupSec = int.Parse(args[argIndex++]);
+                        }
+                        break;
+                    case "--orca_speech_rate":
+                        if (++argIndex < args.Length)
+                        {
+                            config.OrcaSpeechRate = float.Parse(args[argIndex++]);
+                        }
+                        break;
+                    case "--porcupine_sensitivity":
+                        if (++argIndex < args.Length)
+                        {
+                            config.PorcupineSensitivity = float.Parse(args[argIndex++]);
+                        }
+                        break;
+                    case "--short_answers":
+                        if (++argIndex < args.Length)
+                        {
+                            config.ShortAnswers = bool.Parse(args[argIndex++]);
+                        }
+                        break;
+                    case "--profile":
+                        if (++argIndex < args.Length)
+                        {
+                            config.Profile = bool.Parse(args[argIndex++]);
+                        }
+                        break;
+                    case "--help":
+                    case "-h":
+                        Console.WriteLine(HELP_TEXT);
+                        System.Environment.Exit(0);
+                        break;
+                    default:
+                        argIndex++;
+                        break;
+                }
+            }
+
+            if (config.AccessKey == null || config.PicollmModelPath == null)
+            {
+                Console.WriteLine("Access key and LLM model path are required.");

```

Comment:
better to use the arg names
```suggestion
                Console.WriteLine("`access_key` and `picollm_model_path` are required.");

```